gokart/parameter.py

```diff
@@ -5,11 +5,21 @@
 import luigi
 from luigi import task_register
 
+import gokart
+
 logger = getLogger(__name__)
 
 
 class TaskInstanceParameter(luigi.Parameter):
 
+    def __init__(self, *args, **kwargs):
+        bound = kwargs.pop('bound', gokart.TaskOnKart)
```

Comment:
If you mean subtype bound, how about just use 'type' keyword? There might be both upper and lower for 'bound' keyword.